### PR TITLE
Support workflow improvement, quick copy of user charts

### PR DESCRIPTION
**Pull request by elanals**
A pain point in Support is always having to navigate to the publish step in order to duplicate a user chart. This PR adds a quick link in the '**this chart doesn't belong to you**' alert, that:
- duplicates the chart
- places the copy in admin's My Charts
- opens the new chart in another tab on the same editor page.

image

Also got rid of the translation pipe because it wasn't getting used anyway, and it's not really important to have this translated right now.

### CODE REVIEW

File: templates/chart/base.twig [hunk 1]
```diff
@@ -80,6 +79,43 @@ dw.backend.ready(function() {
...
+    $('a#duplicate-chart').on('click', function(e){
+        e.preventDefault();
+        $('i#copying-chart').addClass('fa-spinner fa-spin');
+        $.ajax({
+            type: "POST",
+            url: '/api/charts/'+chart.get('id')+'/copy',
+            xhrFields: {
+                withCredentials: true
+            },
+            success: moveChart,
```
Comment (davidkokkelink) [inline templates/chart/base.twig:91]: is `move chart` even needed? if admins copy, it's [sent to my-charts anyway](https://github.com/datawrapper/datawrapper/blob/master/lib/core/build/classes/datawrapper/ChartQuery.php#L114-L120)

File: templates/chart/base.twig [hunk 2]
```diff
@@ -80,6 +79,43 @@ dw.backend.ready(function() {
...
+            dataType: 'json'
+        });
+    })
+
+    function moveChart(res) {
+        var id = res.data.id;
+        var api = window.location.protocol+'//'+window.location.host.replace('app','api');
```
Comment (davidkokkelink) [inline templates/chart/base.twig:98]: fyi `dw.backend.__api_domain`

File: templates/chart/base.twig [hunk 3]
```diff
@@ -36,8 +36,7 @@
 
 {% if user != chart.user and user.isAdmin() %}
 <div class="alert alert-warning" style="text-align:center;margin-top:10px;">
- {{ "This chart belongs to <a href='/admin/charts/%uid'>%email</a>. Great power comes with great responsibility, so be careful with what you're doing!" | trans
-    | replace({"%email": chart.user.email, "%uid": chart.user.id }) | raw }}
+ {{ "This chart belongs to <a href='/admin/charts/%uid'>%email</a>. Great power comes with great responsibility, so be careful with what you're doing! <a id='duplicate-chart' href=''>Copy Chart</a> <i id='copying-chart' class='fa'></i>" | replace({"%email": chart.user.email, "%uid": chart.user.id }) | raw }}
```
Comment (davidkokkelink) [inline templates/chart/base.twig:39]:
```suggestion
 {{ "This chart belongs to <a href='/admin/charts/%uid'>%email</a>. Great power comes with great responsibility, so be careful with what you're doing, or <a id='duplicate-chart' href=''>copy the chart to your account</a> <i id='copying-chart' class='fa'></i>!" | replace({"%email": chart.user.email, "%uid": chart.user.id }) | raw }}
```

File: templates/chart/base.twig [hunk 4]
```diff
@@ -80,6 +79,25 @@ dw.backend.ready(function() {
...
+        $.ajax({
+            type: "POST",
+            url: '/api/charts/'+chart.get('id')+'/copy',
+            xhrFields: {
+                withCredentials: true
+            },
+            success: openChart,
+            dataType: 'json'
+        });
+    })
```
Comment (davidkokkelink) [inline templates/chart/base.twig:94]:
```suggestion
{% if user != chart.user and user.isAdmin() %}
    $('a#duplicate-chart').on('click', function(e){
        e.preventDefault();
        $('i#copying-chart').addClass('fa-spinner fa-spin');
        $.ajax({
            type: "POST",
            url: '/api/charts/'+chart.get('id')+'/copy',
            xhrFields: {
                withCredentials: true
            },
            success: openChart,
            dataType: 'json'
        });
    });
 {% endif %}
```

**Comment by elanals** [comment]
Alright, all suggestions and requests implemented.

**Pull request merged**